unix/capnp_rpc_unix.ml

```diff
@@ -30,7 +30,7 @@ let parse_uri s =
 module Cap_file = struct
   let load_uri path =
     try
-      let ch = open_in path in
+      let ch = open_in_bin path in
```

Comment:
This seems reasonable, but I'm surprised it's needed. Depending on the platform, we should end up with either `foo\n` or `foo\r\n`, and either way the following `String.trim` should sort it out. What goes wrong on Windows?